python/cudf/cudf/core/column/interval.py

```diff
@@ -80,12 +85,31 @@ def copy(self, deep=True):
         struct_copy = super().copy(deep=deep)
         return IntervalColumn(
             size=struct_copy.size,
-            dtype=cudf.core.dtypes.IntervalDtype(
-                struct_copy.dtype.fields["left"], closed
-            ),
+            dtype=IntervalDtype(struct_copy.dtype.fields["left"], closed),
             mask=struct_copy.base_mask,
             offset=struct_copy.offset,
             null_count=struct_copy.null_count,
             children=struct_copy.base_children,
             closed=closed,
         )
+
+    def as_interval_column(self, dtype, **kwargs):
+        if isinstance(dtype, str) and dtype == "interval":
+            return self
+
+        if isinstance(dtype, (IntervalDtype, cudf.core.dtypes.IntervalDtype)):
```

Comment:
They are slightly different. If the dtype is the string `"interval"` it will not have the attribute `closed` and will throw an error because of this. This is why I had initially separated them. 

I see that this could lead to some unitentional errors, so I've switched out the `isinstance` statements here for `is_interval_dtype(dtype)`, and then using a  try, except block to handle for the error in the case of the string dtype. Let me know if there's a better way I should go about this :D